### ISSUE TITLE
How can I change background image in python wrap.

### ISSUE THREAD

**Issue by Wisgon** [andrewssobral/bgslibrary#136]
Hi, @andrewssobral . Thanks for the great job!
I use python to run the code, and it's cool,  but I wonder to know that when I want to change the background(such as when there is less and less light, I have to change the background), how can I do ?

**Comment by andrewssobral** [comment]
Hi @Wisgon ,
I'm not sure to understand your question, but why you want to change the background?
Do you want to manually update the background image? 
Usually, for most bgs algorithms, the background image is updated automatically...as I understood, you want to fix the background image when it's created (e.g. stopping the update)?
Thanks

**Comment by Wisgon** [comment]
Yes, I want to fix the background image, because the light of the environment is changing, when I use opencv's algorithms, I can manually update background image, so I want to know how to update background image, or it's updated automatically?

**Comment by andrewssobral** [comment]
Yes, usually the background model is updated automatically, but for some algorithms, you can stop the update by setting the learning rate to zero, for example:
AdaptiveBackgroundLearning, DPWrenGA, KDE - parameter alpha
AdaptiveSelectiveBackgroundLearning - parameter alphaLearn
LBAdaptiveSOM, LBFuzzyAdaptiveSOM, LBFuzzyGaussian - parameter learningRate
...
which bgs method are you using?
Thanks

**Comment by AOOOOOA** [comment]
> I use python wrapper, as I know, python wrapper only implement two method "apply" and "getBackgroundModel", so I figure it out with re-init the object like "bgs = libbgs.SuBSENSE()" to get a new "bgs" that have different background.

Hi @andrewssobral 
  I just wondering does other bgs algorithms have learning rate that I can change?

Thank you!
